LiskHQ/lisk-hub
# Implement storing BTC transactions data - Closes #1945

### What issue have I solved?

#1945

### How have I implemented/fixed it?

I changed `getTransactions` API util to be token-agnostic and to get token-specific `getTransactions` (BTC or LSK) based on whether the passed `address` parameter is BTC or LSK address.

Analogically, I changed `getSignleTransaction` to be token-agnostic based on whether the passed `id` parameter is BTC or LSK id.

Note that currently it is based on branch `1946-logic-for-btc-peer`. After that branch PR is merged, I will change the base branch of this PR to `development`

### How has this been tested?

1. Go to  https://jenkins.lisk.io/test/lisk-hub/PR-1955/
2. log it to testnet
2. Set `localStorage.setItem('btc', true);`
1. Go to  https://jenkins.lisk.io/test/lisk-hub/PR-1955/#/explorer/accounts/n3aZt7uZhnBeC9quq6btKyC8qXvskEiE1B
3. see there are BTC transactions loaded.
1. Go to https://jenkins.lisk.io/test/lisk-hub/PR-1955/#/explorer/transactions/553787aa91a55e0385beb81a0a549f02cb3be8492b452fe8b09c1729165a8362
3. see there a BTC transaction details.

Note that there will be another ticket to make the page show everything properly.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
